serving/samples/helloworld-java/service.yaml

```diff
@@ -9,7 +9,4 @@ spec:
       revisionTemplate:
         spec:
           container:
-            image: docker.io/{username}/helloworld-java
-            env:
-            - name: TARGET
-              value: "Spring Boot Sample v1"
+            image: docker.io/hpacodeit/helloworld-java
```

Comment:
change to {username}?

Reply:
It should be different, but not sure I agree w/ {username} unless that is the standard in this repo, since the expansion isn't being done by kubectl.

Reply:
the original which this is based off of uses `{username}`